static/app/utils/useParams.tsx

```diff
@@ -42,6 +43,7 @@ type ParamKeys =
   | 'release'
   | 'relocationUuid'
   | 'replaySlug'
+  | 'ruleId'
```

Comment:
unrelated to this change, but looks like detectorId is in here twice, and workflow id is not (if we want to add it for future proofing)